tweeter.py

```diff
@@ -85,5 +85,23 @@ def tweet():
     tweet = "{} {}".format(antecedent_list[ant_index].format(tweet_subj), postcedent_list[post_index])
     return tweet
 
+@app.route("/stats")
+def stats():
+    adj_index = len(adj_list)
```

Comment:
These variable are better named to indicate that they store length values, i.e, `adj_len` instead of `adj_index`